src/utils/global.ts

```diff
@@ -132,3 +133,12 @@ export function getDescription(
       description.label.toLowerCase() === key,
   )?.data : _.get(descriptions, key)
 }
+
+// Check if file input is of given type - throw error if not
+export function checkInput(guessOptions: { data: string, filename: string }, desiredType: string, desiredFormat: string): void {
+  const detectedType = fingerprint({data: guessOptions.data, filename: convertFullPathToFilename(guessOptions.filename)})
+  if (!(detectedType === desiredType))
```

Comment:
can we swap to using enums in the fingerprinting section for all these string constants for desired type/format?  it'll confer the typical advantages (less likelihood of typos, single source of truth, etc)

Reply:
Should this be an hdd-converters change or a SAF-CLI change only?